java/pending_code_changes.txt

```diff
@@ -1 +1,11 @@
+Pending metadata changes:
+* Metadata fixes for BY, KR, NA
```

Comment:
Also the release notes end up without a blank line between different types of sections, although this is also confusing for the PR description :) https://github.com/googlei18n/libphonenumber/blob/642590c597a49b526e632dc90770fc74b8a794ea/java/release_notes.txt
b/28013360

Just FYI. We should fix at some point.